src/state-summary/state-card-content.html

```diff
@@ -13,40 +13,40 @@
 <link rel="import" href="state-card-weblink.html">
 
 <script>
-Polymer({
-  is: 'state-card-content',
-
-  properties: {
-    hass: {
-      type: Object,
-    },
-
-    inDialog: {
-      type: Boolean,
-      value: false,
+  Polymer({
```

Comment:
Indeed, seems my editor has wrong indentation set.
If I remember correctly there should be no changes in this file anymore, since at first I added `state-card-weather`, which is not used anymore.
I'll remove this asap.